sway/commands.c

```diff
@@ -472,9 +471,7 @@ struct cmd_results *config_commands_command(char *exec) {
 	if (!policy) {
 		policy = alloc_command_policy(cmd);
 		sway_assert(policy, "Unable to allocate security policy");
```

Comment:
`sway_assert` is not designed not to always be an assert. We should probably do this:

```c
if (!sway_assert(policy, "Unable to allocate security policy")) {
    goto cleanup;
}
```

Reply:
probably